src/controllers/RatingController.js

```diff
@@ -1,6 +1,6 @@
 import status from '../config/status';
 
-import { Article } from '../queries';
+import { Article, Rating } from '../queries';
```

Comment:
@sengayire , as ratings are for articles, I would suggest to import it in articles queries and use it as the following : `Article.rate.add()` ,`Article.rate.remove()` or `Article.rate.sort()`, on my understanding it's more logic as we can have a rating of something else, and in that case it can be confusing about which resource we want to rate

Reply:
good ideal, I am working on that

Reply:
resolved